include/Page/User.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Bulk Move Users admin page.
+ *
+ * @since 2.0.0
+ */
+class BM_Page_User extends BM_Page_Base {
+
+	protected function initialize() {
+		$this->slug            = 'bulk-move-users';
+		$this->page_title      = __( 'Bulk Move Users', 'bulk-move' );
+		$this->menu_title      = __( 'Bulk Move Users', 'bulk-move' );
+		$this->warning_message = __( 'WARNING: Posts moved once cannot be reverted. Use with caution.', 'bulk-move' );
+		$this->capability      = 'edit_others_posts';
+	}
+
+	public function register() {
+		parent::register();
+
+		add_filter( 'bm_plugin_action_links', array( $this, 'append_to_plugin_action_links' ) );
```

Comment:
I think it makes sense to move filter and function this to base class.

Reply:
> I think it makes sense to move filter and function this to base class.

Fixed.

Reply:
Is this also fixed? I don't see the changes.